docs/run_in_existing_vpc.md

```diff
@@ -63,7 +63,7 @@ but it's better to avoid the later confusion!)
 
 ## Running in a shared subnet
 
-You can also use a shared subnet. Doing so is not recommended unless you are using external networking.
+You can also use a shared subnet. Doing so is not recommended unless you are using external networking (weave or flannel or calico or kope-routing etc).
```

Comment:
Thanks for the PR

Sorry to be picky, we need to make kope-routing "external" and the others CNI. Do you want to fix that or have is fix that later? I know you did not write the initial docs